pkg/controller/operators/catalog/operator_test.go

```diff
@@ -1119,6 +1167,15 @@ func objectReference(name string) *corev1.ObjectReference {
 	return &corev1.ObjectReference{Name: name}
 }
 
+func yamlFromFilePath(fileName string) string {
+	yaml, err := ioutil.ReadFile(fileName)
+	if err != nil {
```

Comment:
I understand what you are doing here but I wonder what is expected result of the test case if the error occurs here and an empty string is returned? I think you should return the error and handle it properly in the test case to avoid some false positive in case the test data is moved/removed accidentally.

Reply:
Seems to be a copy of https://github.com/operator-framework/operator-lifecycle-manager/pull/1227#discussion_r369936379